install.js

```diff
@@ -53,29 +53,26 @@ if (parentPackage &&
 }
 
 if (process.env.CLDR_URL) {
-  srcUrl = srcUrl.replace(
-    "http://www.unicode.org/Public/cldr",
-    process.env.CLDR_URL.replace(/\/$/, "")
-  );
-
+  // Load custom urls.json file from env variable.
+  srcUrl = process.env.CLDR_URL;
```

Comment:
@ccschneidr and it has nothing to do, right? Am I missing something? In case you use CLDR_URL, its value is used instead of the various URLs from the json file, i.e., the URLs from the json file are completely ignored.

Having said that, I would be happy to accept a change that does what you mean. Let's say we name it `CLDR_BASE_URL`, it would replace the base url from the various URLs with the passed base URL.